mmhuman3d/models/body_models/smplx.py

```diff
@@ -371,5 +382,431 @@ def forward(self,
             output['vertices'] = smplx_output.vertices
         if return_full_pose:
             output['full_pose'] = smplx_output.full_pose
+        return output
+
+
+@dataclass
+class ModelOutput(SMPLXOutput):
+    smpl_joints: Optional[torch.Tensor] = None
+    joints_J19: Optional[torch.Tensor] = None
+    smplx_vertices: Optional[torch.Tensor] = None
+    flame_vertices: Optional[torch.Tensor] = None
+    lhand_vertices: Optional[torch.Tensor] = None
+    rhand_vertices: Optional[torch.Tensor] = None
+    lhand_joints: Optional[torch.Tensor] = None
+    rhand_joints: Optional[torch.Tensor] = None
+    face_joints: Optional[torch.Tensor] = None
+    lfoot_joints: Optional[torch.Tensor] = None
+    rfoot_joints: Optional[torch.Tensor] = None
+
+
+class SMPLXLayer_PyMAFX(_SMPLXLayer):
```

Comment:
What's the difference between `SMPLXLayer_PyMAFX` and `SMPLXLayer`?

Reply:
`SMPLXLayer` doesn't have `def get_global_rotation`, now I have added  it, and deleted `SMPLXLayer_PyMAFX`